react_practice/react-apps/waitlist-page-v1/src/CTAForm.js

```diff
@@ -45,11 +40,10 @@ function CTAForm() {
           value={value}
           onChange={handleChange}
         />
-        <PrimaryButton
+        <PrimaryBtn
           title="Join the waitlist"
           className="cta-form-btn"
-          onKeyPress={handleKeyPress}
```

Comment:
Looking at the definition for the `PrimaryBtn` component, it didn't look like you did anything with [`onKeyPress`]( https://github.com/tyreljackson-L904/React_Dev/pull/1/files#diff-416ab0255a17863d9fe345d4dc497ff283c70f3035b6642aea1432d76f1a313dL3), so this was essentially a dead-end at this point.

Keep in mind that when you pass a prop to a component, it will do nothing unless you use it there. Looking at `PrimaryBtn`, you were only consuming `title` and `handleSubmit`, so this function would not fire. I went ahead and removed it as the `onSubmit` on the form covers the enter key.